[2022-03-15 19:34:01,474] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 manual__2022-03-15T19:33:51.377489+00:00 [queued]>
[2022-03-15 19:34:01,622] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 manual__2022-03-15T19:33:51.377489+00:00 [queued]>
[2022-03-15 19:34:01,623] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:34:01,623] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-03-15 19:34:01,623] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 19:34:01,699] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_3> on 2022-03-15 19:33:51.377489+00:00
[2022-03-15 19:34:01,849] {standard_task_runner.py:52} INFO - Started process 12997 to run task
[2022-03-15 19:34:01,896] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_3', 'manual__2022-03-15T19:33:51.377489+00:00', '--job-id', '160', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule.py', '--cfg-path', '/tmp/tmpej726uzh', '--error-file', '/tmp/tmpup86_s2u']
[2022-03-15 19:34:01,940] {standard_task_runner.py:77} INFO - Job 160: Subtask task_3
[2022-03-15 19:34:02,548] {logging_mixin.py:109} INFO - Running <TaskInstance: trigger_rule.task_3 manual__2022-03-15T19:33:51.377489+00:00 [running]> on host airflowvm
[2022-03-15 19:34:03,271] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T19:33:51.377489+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-15T19:33:51.377489+00:00
[2022-03-15 19:34:03,277] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-15 19:34:03,280] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'exit 0']
[2022-03-15 19:34:03,372] {subprocess.py:85} INFO - Output:
[2022-03-15 19:34:03,394] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 19:34:03,639] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_3, execution_date=20220315T193351, start_date=20220315T193401, end_date=20220315T193403
[2022-03-15 19:34:03,869] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 19:34:04,499] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
